### ISSUE TITLE
Problem calling main class

### ISSUE THREAD

**Issue by gaBrisingr06** [kitao/pyxel#580]
I created my class App and into it I created a variable "map" and then for a function I wanted to call "map". However it doesn't recognized "map" saying that "App has no attribute "map" ". So I was wondering if there's an alternative or a way to do it I don't know.

Thank you in advance

**Comment by merwok** [comment]
Can you show your code?

**Comment by gaBrisingr06** [comment]
![20241115_175753](https://github.com/user-attachments/assets/fb93631e-9df9-43b7-aaa9-e1be3fbee9fd)
![20241115_175722](https://github.com/user-attachments/assets/a2959d4d-f797-4b82-9ed8-d3131b264e34)

"Celeste" is the name of the App class and yes self.map is in this class

**Comment by merwok** [comment]
Instances have attributes defined in \_\_init\_\_, but the class itself does not

**Comment by gaBrisingr06** [comment]
Yes so I tried to create an instance like that instead of just calling the class :

celeste = Celeste()
celeste

But it didn't worked when I called celest
So I just made map global, it took me less time

**Comment by kitao** [comment]
This issue looks a Python usage problem rather than a Pyxel's issue. So let me close this issue.
If you have some information which indicates this issue is because of Pyxel, please make a new issue.
